### PR TITLE
DWW OKTA-106838 Update pages for Users and Apps with new info about s…

### PR DESCRIPTION
**Pull request by daytonwilliams-okta**
…pecial handling of Image Properties

@brianzuzga-okta @mystiberry-okta Please review my doc PR for the profile image placeholder URL changes

### CODE REVIEW

File: _source/_docs/api/resources/users.md [hunk 1]
```diff
@@ -105,6 +105,25 @@ The User model defines several **read-only** properties:
 | _links                | [link relations](#links-object) for the user's current `status` | [JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06)                                      | TRUE     | FALSE  | TRUE     |           |           |            |
 |-----------------------+-----------------------------------------------------------------+-----------------------------------------------------------------------------------------------------+----------+--------+----------+-----------+-----------+------------|
 
+{% beta %}
+
+Currently, while the profile image feature is in beta, when inspecting the schema, image properties are the `Object` datatype, containing an `extendedType` field with the value `Image`.
```
Comment (brianzuzga-okta) [inline _source/_docs/api/resources/users.md]: Trying to reduce the wordiness:  During the profile image beta, image property definitions in the schema are of the `Object` data type with an additional `extendedType` of `Image`.

File: _source/_docs/api/resources/users.md [hunk 2]
```diff
@@ -105,6 +105,25 @@ The User model defines several **read-only** properties:
...
+When a user is retrieved via the API, however, the value will be a URL (represented as a String).  Some caveats apply:
+
+1) Image properties are described differently in the schema than how the data actually comes back.  The precise way this is signaled may change; however, the discrepancy is deliberate.  An image property is similar to a URL (in that the value is the same type), but is not a URL property. Special handling rules apply (described below).
```
Comment (brianzuzga-okta) [inline _source/_docs/api/resources/users.md]: "This discrepancy is deliberate for the time being, but is likely to change after beta." ?

File: _source/_docs/api/resources/apps.md [hunk 3]
```diff
@@ -3864,6 +3864,25 @@ Your request will be rejected with a `403 Forbidden` status for applications wit
 
 > The Okta API currently doesn't support entity tags for conditional updates.  It is only safe to fetch the most recent profile with [Get Assigned User for Application](#get-assigned-user-for-application), apply your profile update, then `POST` back the updated profile as long as you are the **only** user updating a user's application profile.
 
+{% beta %}
+
+Currently, while the profile image feature is in beta, when inspecting the schema, image properties are the `Object` datatype, containing an `extendedType` field with the value `Image`.
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/apps.md]: Capitalize Beta everywhere.

File: _source/_docs/api/resources/apps.md [hunk 4]
```diff
@@ -3864,6 +3864,25 @@ Your request will be rejected with a `403 Forbidden` status for applications wit
...
+When a user's app profile is retrieved via the API, however, the value will be a URL (represented as a String).  Some caveats apply:
+
+1) Image properties are described differently in the schema than how the data actually comes back.  The precise way this is signaled may change; however, the discrepancy is deliberate.  An image property is similar to a URL (in that the value is the same type), but is not a URL property. Special handling rules apply (described below).
+
+2) During beta, the URL returned is a placeholder URL, resolving to a placeholder image.  By GA, the URL returned will resolve to the image (that is, a logged in user can click it and retrieve the image).
+
+**Updating image property values via Users API**
+
+Okta does not support uploading images via the Apps API.  As such, all operations in this API that update properties on a user work in a slightly different way when applied to image properties:
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/apps.md]: Suggest we delete "As such"

File: _source/_docs/api/resources/users.md (hunk 1 above)
Comment (mystiberry-okta) [inline _source/_docs/api/resources/users.md]: Agree.

File: _source/_docs/api/resources/users.md (hunk 2 above)
Comment (mystiberry-okta) [inline _source/_docs/api/resources/users.md]: Agree (don't forget to capitalize Beta).

File: _source/_docs/api/resources/users.md (hunk 1 above)
Comment (daytonwilliams-okta) [inline _source/_docs/api/resources/users.md]: Done.

File: _source/_docs/api/resources/users.md (hunk 2 above)
Comment (daytonwilliams-okta) [inline _source/_docs/api/resources/users.md]: Done.

File: _source/_docs/api/resources/apps.md (hunk 3 above)
Comment (daytonwilliams-okta) [inline _source/_docs/api/resources/apps.md]: Done.

File: _source/_docs/api/resources/apps.md (hunk 4 above)
Comment (daytonwilliams-okta) [inline _source/_docs/api/resources/apps.md]: Done.

**Comment by mystiberry-okta** [comment]
Looks great! Can I merge this now, @daytonwilliams-okta ?

**Comment by daytonwilliams-okta** [comment]
As long as @brianzuzga-okta is okay with that, I am okay with you merging this, @mystiberry-okta 

Thanks for your assistance with this.

**Comment by brianzuzga-okta** [comment]
:ship:

**Pull request merged**